experiences/multicard/multicard.test.js

```diff
@@ -0,0 +1,81 @@
+const { bootstrapExperience } = require("../../tests/utils/bootstrap");
```

Comment:
What's the differenec between `multicard.test.js` and `multicard.localhost.test.js` ? This file seems duplicated?

Reply:
Its outdated, I'll remove it for now, we are focusing on the homepage experience.